cmd/snap-bootstrap/main.go

```diff
@@ -66,6 +66,9 @@ func parseArgs(args []string) error {
 	p := parser()
 
 	_, err := p.ParseArgs(args)
+	if err != nil {
+		logger.Noticef("execution error: %v", err)
```

Comment:
Silly question - why is this added/needed? Our default logger logs to os.Stderr iirc and on error parseArgs will also fmt.Fprintf() to os.Stderr and exit 1 ? What am I missing :)

Reply:
Great question in fact ;). I added this because otherwise you would not see the error in the console. Writing to os.stderr is not enough, you need to write to the journal so journald can output when configured to do so.